assets/js/modules/analytics/components/setup/SetupMain.js

```diff
@@ -78,7 +79,7 @@ export default function SetupMain( { finishSetup } ) {
 	let viewComponent;
 	// Here we also check for `hasResolvedAccounts` to prevent showing a different case below
 	// when the component initially loads and has yet to start fetching accounts.
-	if ( isDoingSubmitChanges || ! hasResolvedAccounts || isNavigating ) {
+	if ( isDoingSubmitChanges || ! hasResolvedAccounts || isNavigating || setupFlowMode === undefined ) {
```

Comment:
We need to update storybook stories to have legacy setup flow mode because right now this change causes all stories to display the progress bar: https://google.github.io/site-kit-wp/storybook/pull/3331/?path=/story/analytics-module-setup--start

Reply:
Nice spot @eugene-manuilov !  :grimacing: 

Next time I'll check storybook.  I'm just assuming that VRTs will fail :joy:  (I've never had them on a project before)

Fix done. You can check the above link again :smile: 

:heart:  the auto deploy of storybook per branch!